subgraphs/balancer-forks/src/prices/common/constants.ts

```diff
@@ -46,6 +46,22 @@ export const WHITELIST_TOKENS_LIST: string[] = [
   "CRV",
 ];
 
+export const USD_PEGGED_TOKENS: string[] = [
```

Comment:
Why did you add these here?

Reply:
*All of the USD_PEGGED_TOKENS.

Reply:
These are all the stable tokens on which the Price lib failed to fetch a correct price.